src/Body.java

```diff
@@ -0,0 +1,16 @@
+public class Body {
+
+    private Colors color;
+
+    Body(String color) {
+        this.color = Colors.valueOf(color.toUpperCase());
+    }
+
+    void clean() {
+        System.out.println("Your " + color.toString() + " car is washed");
```

Comment:
Remove '.toString()' - it is done automatically

Reply:
@malvitvik good idea :)